### PR TITLE
EndAlignment configuration

### PR DESCRIPTION
**Pull request by tommyalvarez**
Enforces style `start_of_line` for `EndAlignment` cop.

### CODE REVIEW

File: config/rubocop-2.5.yaml [hunk 1]
```diff
@@ -142,6 +142,7 @@ Layout/EmptyLinesAroundModuleBody:
 Layout/EndAlignment:
   Description: 'Align ends correctly.'
   Enabled: true
+  EnforcedStyleAlignWith: start_of_line
```
Comment (tommyalvarez) [inline config/rubocop-2.5.yaml]: Are we still mantaining these files? The current and [this](https://github.com/bespokepost/code-climate-config/pull/9/files#diff-58798e70686cfd8f71b322d2edd42738b73caeb596aab641d27a5af292392dc2R82)

File: config/rubocop-2.4.yml [hunk 2]
```diff
@@ -79,6 +79,8 @@ Layout/ConditionPosition:
 Layout/DefEndAlignment:
   Description: 'Align ends corresponding to defs correctly.'
   Enabled: true
+  EnforcedStyleAlignWith: start_of_line
+
```
Comment (jackchoi) [inline config/rubocop-2.4.yml]: Extra new line

File: config/rubocop/1.9.1/rubocop.default.yml [hunk 3]
```diff
@@ -589,7 +589,7 @@ Layout/EndAlignment:
   # situations, `end` should still be aligned with the keyword.
   # The value `start_of_line` means that `end` should be aligned with the start
   # of the line which the matching keyword appears on.
-  EnforcedStyleAlignWith: keyword
+  EnforcedStyleAlignWith: start_of_line
```
Comment (jackchoi) [inline config/rubocop/1.9.1/rubocop.default.yml]: We shouldn't be modifying the default, this is the base template **without** our customizations. Read [this comment](https://github.com/bespokepost/code-climate-config/pull/8#discussion_r628427101)

Make the changes in [this file](https://github.com/bespokepost/code-climate-config/blob/83a2ffb77b2fe6bfea6efcedd74b07e8bdae6b97/config/rubocop/1.9.1/rubocop-2.7.yml) instead.

File: config/rubocop-2.5.yaml (hunk 1 above)
Comment (jackchoi) [reply config/rubocop-2.5.yaml]: These are safe to remove

**Review by jackchoi — CHANGES_REQUESTED** [review]

File: config/rubocop/1.9.1/rubocop.default.yml (hunk 3 above)
Comment (tommyalvarez) [reply config/rubocop/1.9.1/rubocop.default.yml]: Actually seems that I read in the past by the looks of the 👍  I left there 😄 .

**Review by jackchoi — APPROVED** [review]

**Pull request merged**